src/viam/examples/modules/complex/gizmo/api.cpp

```diff
@@ -22,8 +22,8 @@ GizmoRegistration::GizmoRegistration(const google::protobuf::ServiceDescriptor*
     : ResourceRegistration(service_descriptor){};
 
 std::shared_ptr<ResourceServer> GizmoRegistration::create_resource_server(
-    std::shared_ptr<ResourceManager> manager) {
-    return std::make_shared<GizmoServer>(manager);
+    std::shared_ptr<ResourceManager> manager, std::shared_ptr<Server> server) {
+    return std::make_shared<GizmoServer>(manager, server);
```

Comment:
[nit]: `std::move(manager), std::move(server)`. You can argue that it makes no difference here and that'd be pretty true, but its a good habit to be consistent about using move. There's a lot of refcount jiggling happening here for basically no reason.